packages/core/src/root_files/localstorage-sync.html

```diff
@@ -16,6 +16,7 @@
             var allowed_origins = [
                 `https://smarttrader.${domain_url}`,
                 'https://smarttrader-staging.deriv.com',
```

Comment:
Shouldn’t we remove `smarttrader-staging.deriv.com`?

Reply:
Since we had both of them in other places in smart-trader I didn't remove it here as well for consistency.
Who could we ask?

Reply:
```suggestion
                'https://smarttrader-staging.${domain_url}',
```